episodes/2-keras.Rmd

```diff
@@ -121,48 +109,15 @@ penguins.head()
  | 3 | Adelie | Torgersen | NaN  | NaN  | NaN   | NaN    | NaN    |
  | 4 | Adelie | Torgersen | 36.7 | 19.3 | 193.0 | 3450.0 | Female |
 
+ All columns but the 'species' columns are features that we can use.
+
+There are 344 samples and 7 columns, so 6 features:
  ```python
- penguins.describe()
+ penguins.shape
+ ```
+ ```output
+ (344, 7)
  ```
- 
-
- |       | bill_length_mm | bill_depth_mm | flipper_length_mm | body_mass_g |
- |------:|---------------:|--------------:|------------------:|------------:|
- | count |     342.000000 |    342.000000 |        342.000000 |  342.000000 |
- |  mean |      43.921930 |     17.151170 |        200.915205 | 4201.754386 |
- |   std |       5.459584 |      1.974793 |         14.061714 |  801.954536 |
- |   min |      32.100000 |     13.100000 |        172.000000 | 2700.000000 |
- |   25% |      39.225000 |     15.600000 |        190.000000 | 3550.000000 |
- |   50% |      44.450000 |     17.300000 |        197.000000 | 4050.000000 |
- |   75% |      48.500000 |     18.700000 |        213.000000 | 4750.000000 |
- |   max |      59.600000 |     21.500000 |        231.000000 | 6300.000000 |
-
-**2.** We can get the unique values in the `species` column using the `unique` function of pandas.
-It shows the target class is stored as a string and has 3 unique values. This type of column is
-usually called a 'categorical' column.
-```python
-penguins["species"].unique()
-```
-```output
-array(['Adelie', 'Chinstrap', 'Gentoo'], dtype=object)
-```
-
-**3.** Using `describe` function on the species column shows there are 344 samples
-unique species
-```python
-penguins["species"].describe()
-```
-
-```output
-count        344
-unique         3
-top       Adelie
-freq         152
-Name: species, dtype: object
-```
-
-::::
-:::
 
 ### Visualization
```

Comment:
Pairplot needs to be in a new line in order to render it as a separate subsection.

Reply:
Check